DQM/L1TMonitor/python/L1TCaloLayer1Summary_cff.py

```diff
@@ -19,4 +9,4 @@
 simCaloStage2Layer1Digis.ecalToken = cms.InputTag("ecalDigis", "EcalTriggerPrimitives")
 simCaloStage2Layer1Digis.hcalToken = cms.InputTag("hcalDigis", "")
```

Comment:
Just for my understanding, why is it okay to let this DQM cff modify `simCaloStage2Layer1Digis` ?

Reply:
The original `simCaloStage2Layer1` is fed from simulated TPs. For emulation purposes here, it is likely better to have them fed from unpacked TPs, and look at the emulation results from that.

Reply:
Is this a separate instance used solely for DQM purposes, or is this DQM cff modifying the results of the L1T emulation itself (similarly to what happened with the bug that this PR is trying to fix) ?

Reply:
This sequence _should_ only be for DQM instances, but since it isn't cloned, it is likely causing the same problem, now that you mention it. I'll fix that.